dev/MRTCore/mrt/mrm/UnitTests/DecisionInfo.UnitTests.cpp

```diff
@@ -84,12 +84,7 @@ void DecisionInfoUnitTests::SimpleBuilderTests()
         return;
     }
 
-    // Add test data to builders
-    // Although it is tempting to remove the '!' below to address the PREfast issue, doing that broke the build. 
```

Comment:
I guess we'll see if the azp run fails... hopefully not. 😊🤞

Reply:
It's not clear to me why it would fail. :)